src/extractPdfText.ts

```diff
@@ -1,51 +1,59 @@
 import { getPdfPagesCount } from "./getPdfPagesCount";
 import { capitalizeFirstLetter } from "./helperFunctions";
-const pdfLib = require('pdf-to-text');
-// figure out how to deal with this import vs require thing
```

Comment:
This error is simply telling you that there are no `types` defined for `pdf-to-text`. Seeing as `pdf-to-text` doesn't ship with types, its safe to ignore